### ISSUE TITLE
Locomotive not loading full page at first load on CHROME

### ISSUE THREAD

**Issue by lucasmpramos** [locomotivemtl/locomotive-scroll#412]
Hi there guys.

I'm working on a website https://mm18-arquitetura.webflow.io

After I implemented Locomotive scroll just for basic scroll smoothing, the page doesn't load 100%. It cuts down on the middle of the image above the footer.

<img width="1223" alt="image" src="https://user-images.githubusercontent.com/50158449/160193885-80ea9104-90cd-4b78-a7a3-51b8d74d3847.png">

After another refresh (without clearing cache) it shows the full page. 

Thank you!

**Comment by mihau1** [comment]
Exacly the same issue in my case. Footer sections is not visible - only after resize browser window... 
but sometimes after resize i have 300px more on the bottom of the page.
Very strange

**Comment by roojai** [comment]
Same issue!

**Comment by dimndot** [comment]
Any idea how to rectify it?

**Comment by roojai** [comment]
YEs I found a few other threads with the same issue. Seems related to the content being loaded after initialization (eg lazy loaded images).  Locomotive needs to recalculate the page length. To do this you need to call update() on the locomotive instance.

**Comment by crissrocca** [comment]
I use this solution, but it's a patch that I need to call it all the time when I reload the pages. It isn't a locomotive js library solution.

```
import LocomotiveScroll from 'locomotive-scroll';
const scroller = new LocomotiveScroll({
  el: document.querySelector('[data-scroll-container]'),
  smooth: true
});

const imagesLoaded = require("imagesloaded"); // import the library or can use cdn
const loader = document.getElementById("loading");
const add_class_images_loaded = () => loader.classList.add("d-none");

imagesLoaded( document.querySelector('body'), function( instance ) {
  console.log('all images are loaded');
  scroller.update();
  add_class_images_loaded();
});
```

A few days ago I opened a comment about this, to see another solution [https://github.com/locomotivemtl/locomotive-scroll/issues/476](issue)

**Comment by able2create** [comment]
I´m using it on a Wordpress Project (OnePager). Nothing fancy is going on, just a few direction settings, parallax and a "video stop playing" as user scrolls stuff. 

As a lot of people mention - my issue was the "cut off" one.
I have a footer, which could not be seen. 
If I resized the window - everything works "fine". The Footer was there and a huge extra space after it.

All the js for "waiting", "update" or "timeout" didn´t work for me.
I realized, that the issue was gone, when I deleted the js slider.
So I put in a min-height on the slider and everything worked great.

It is the height of the content being calculated - before all the content is loaded.

For me and for now it is working.

Maybe some people will help this.

**Comment by sirhenryhu** [comment]
Sam here. The footer section is interrupted before it loads completely. The issue disappears once you reload/resize the page which is not ideal for visitors. I would appreciate any workaround.

**Comment by robinsxv** [comment]
Still having this problem 👆 any solutions yet?

**Comment by equinoxe3d** [comment]
Had this issue on a WordPress site (scroll stopping mid-page in Chrome), and in my case it was not caused directly by Locomotive but by a lazy rendering feature within the WP Rocket cache plugin. It was lazy loading sections of the page asynchronously, thus breaking the initial scroll calculations for Locomotive. Emptying the Priority Elements cache solved it for me, and if it ever comes back there's a way to exclude HTML elements from it : https://docs.wp-rocket.me/article/1835-automatic-lazy-rendering
